### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by Jubeki** [squirephp/squire#1] [Ideas]
Do you want translations for countries and capital cities?
I would use a translator API (Probably Google Translator) to translate them into different languages.
The translations would be available in a <language>.json file. These files could be published to the user or could be loaded directly in the ServiceProvider.

**Comment by danharrin** [comment]
Interesting, we could always load a different .csv based on the app's locale. The only issue is that Sushi caching would probably break, so we'd need to build our own solution.

I'm not sure that I have the time to translate the data myself, but if anyone would like to support me with that they're more than welcome. If someone puts in that effort, I'll happy to tweak the package to support these translations.

I'll leave this issue open for further discussion and collaboration 👍

**Comment by bonzai** [comment]
For countries and continents we can use CLDR (Common Locale Data Repository) data:

- [English](https://raw.githubusercontent.com/unicode-cldr/cldr-localenames-full/master/main/en/territories.json) (en)
- [German](https://raw.githubusercontent.com/unicode-cldr/cldr-localenames-full/master/main/de/territories.json) (de)
- [...](https://github.com/unicode-cldr/cldr-localenames-full/tree/master/main)

**Comment by Jubeki** [comment]
I will use the CLDR then to start extracting the languages. I will probably write a tool for it.
